frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -4,6 +4,7 @@ import {BoardActionsRegistryService} from "core-app/modules/boards/board/board-a
 import {BoardStatusActionService} from "core-app/modules/boards/board/board-actions/status/status-action.service";
 import {BoardVersionActionService} from "core-app/modules/boards/board/board-actions/version/version-action.service";
 import {QueryUpdatedService} from "core-app/modules/boards/board/query-updated/query-updated.service";
+import {BoardAssigneeActionService} from "core-app/modules/boards/board/board-actions/assignee/assignee-action.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/078af2d37f20cb568fc50eccd2f70f7cc8bceb6d/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L8"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>